server/graphql/v2/mutation/OrderMutations.js

```diff
@@ -315,6 +320,42 @@ const orderMutations = {
           throw new ValidationFailed(`Only pending/expired orders can be marked as paid, this one is ${order.status}`);
         }
 
+        if (args?.details) {
+          const { totalAmount, paymentProcessorFeesAmount, platformTipAmount } = args.details;
+          const host = await toAccount.getHostCollective();
+          if (totalAmount) {
+            if (totalAmount?.currency !== order.currency) {
+              throw new ValidationFailed('totalAmount currency must match expense currency');
+            }
+            order.set('totalAmount', totalAmount.valueInCents || floatAmountToCents(totalAmount.value));
+          }
+          if (paymentProcessorFeesAmount) {
+            if (paymentProcessorFeesAmount?.currency !== host.currency) {
+              throw new ValidationFailed('paymentProcessorFeesAmount currency must match host currency');
```

Comment:
I don't think we should assume paymentProcessorFees to be passed as hostCurrency. It simply needs to be consistent with `order.currency` and should be later converted to `hostCurrency` when inserted in the ledger.

Reply:
In the user story, the host admin is checking the amount in their bank statement, wouldn't be easier to get all the amounts in the host currency? If not, should the host admin be able to pick which currency their passing?
Perhaps I should just consider that any currency is fine and do the calculations in the API, WDYT?